api/src/main/java/com/eleks/academy/pharmagator/configs/WebConfig.java

```diff
@@ -12,4 +18,24 @@
     public ModelMapper modelMapper() {
         return new ModelMapper();
     }
+
+    @Value("${pharmagator.data-providers.pharmacy-anc.url}")
+    private String pharmacyANCBaseUrl;
+
+    @Bean(name = "pharmacyANCWebClient")
+    public WebClient pharmacyANCWebClient() {
```

Comment:
move this to `api/src/main/java/com/eleks/academy/pharmagator/config/DataProvidersConfig.java`